src/content/api/media-uploads.mdoc

```diff
@@ -0,0 +1,46 @@
+---
+title: Media Uploads
+description: Media Uploads API Reference
+draft: true
+nav:
+  path: API
+  order: 7
+---
+
+## Model
```

Comment:
@MeganSteenkamp Are we going to add this heading to the sidebar?

Reply:
@tom-blake yes, thanks.
Have made a separate [task](https://www.notion.so/centrapay/Render-H2-headings-in-sidebar-9baa89eca1274946b12516b67de306a4?pvs=4) for this.